extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+            ]
+          }
+        }
+      },
+      "statistics": {
+        "classes": {
+          "building": {
+            "count": 100000,
+            "properties": {
+              "height": {
+                "min": 3.9,
+                "max": 341.7,
+                "mode": 5.0
+              },
+              "buildingType": {
+                "occurrences": {
+                  "Residential": 50000,
+                  "Commercial": 40950,
+                  "Hospital": 50
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Semantics
+
+By default properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. The full list of built-in semantics can be found in the [Cesium Metadata Semantic Reference](../../../specification/Metadata/Semantics).
+
+Tileset authors may define their own application- or domain-specific semantics separately.
+
+The example below uses two built-in semantics, `NAME` and `ID`, and one custom semantic, `_HEIGHT`.
+
+```jsonc
+{
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "building": {
+            "properties": {
+              "name": {
+                "type": "STRING",
+                "semantic": "NAME"
+              },
+              "id": {
+                "type": "STRING",
+                "semantic": "ID"
+              },
+              "height": {
+                "type": "FLOAT32",
+                "semantic": "_HEIGHT"
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Styling
+
+TODO - requires more experimentation in CesiumJS
+
+TODO - use semantics in styling language
+
+## JSON Schema Reference
+
+* [Tileset extension](#tileset-extension)
+* [Tile extension](#tile-extension)
+* [Content extension](#content-extension)
+* [`3DTILES_implicit_tiling` subtree extension](#3dtiles_implicit_tiling-subtree-extension)
+
+<!-- omit in toc -->
+## Tileset Extension
+* [`Tileset extension`](#reference-3dtiles_metadata-3d-tiles-extension) (root object)
+* [`Schema `](#reference-schema)
+   * [`Class`](#reference-class)
+      * [`Class Property`](#reference-class-property)
+   * [`Enum`](#reference-enum)
+      * [`Enum Value`](#reference-enum-value)
+* [`Tileset Metadata`](#reference-tileset)
+* [`Group Metadata`](#reference-group)
+* [`Statistics`](#reference-statistics)
+   * [`Class Statistics`](#reference-statistics-class)
+      * [`Property Statistics`](#reference-statistics-class-property)
+
+---------------------------------------
+<a name="reference-3dtiles_metadata-3d-tiles-extension"></a>
+<!-- omit in toc -->
+## Tileset extension
+
+3D Tiles extension that defines metadata for a tileset.
+
+**`Tileset extension` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**schema**|`schema`|An object defining classes and enums.|No|
```

Comment:
Can both `schema` and `schemaUri` be defined?  Maybe not because then duplicates would have to be resolved?  But one of the two must be defined?